docs/xmso.md

```diff
@@ -24,12 +24,13 @@
 
 # -Xmso
 
-
-Sets the initial stack size for operating system threads.
+Sets the native stack size for operating system threads.
 
 You can use the `-verbose:sizes` option to find out the values that the VM is currently using.
 
-If you exceed the maximum value, a `java/lang/StackOverflowError` message is reported.
+When your application makes a JNI call, the VM calculates whether the memory required for the call will exceed the `-Xmso` value. If so, a `java/lang/StackOverflowError` error is thrown.
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** This option relates to the native stack only. There is a separate, Java&trade; stack for operating system threads, for which you use different options (see the link in the [See also](#see-also) section).
```

Comment:
> There is a separate, Java&trade; stack for operating system threads

The "for operating system threads" part is incorrect, I expect it should be "There is a separate stack for Java threads".

Reply:
Good spot! Fixed

Reply:
There is 1 thread - it operates either on a java stack or on a native stack.  Maybe phrasing this as:

```
Java methods and native methods run on two different stacks and the VM handles switching between them for JNI calls.  Each stack is sized using separate options.
```